external/asf4-drivers/qtouch/qtouch.h

```diff
@@ -242,13 +242,15 @@ extern "C" {
 #define DEF_NUM_SCROLLERS              2// Number of scrollers (sliders or wheels)
 #define DEF_SCROLLER_NUM_CHANNELS   (DEF_NUM_CHANNELS / DEF_NUM_SCROLLERS)// Assumes same number for each scroller
 #define DEF_SCROLLER_RESOLUTION      256// Scroller resolution in bits
-#define DEF_SCROLLER_DET_THRESHOLD    32// Scroller detect threshold
+#define DEF_SCROLLER_DET_THRESHOLD    25// Scroller detect threshold
+#define DEF_SCROLLER_TOUCH_THRESHOLD    25 // Scroller active threshold
+#define DEF_SCROLLER_UNTOUCH_THRESHOLD    20 // Scroller active threshold
```

Comment:
i guess these may need some trial and error with different fingers...

Reply:
For me (with my device) it worked better with the thresholds 18 and 15. Maybe we should let the user choose, like have a low/medium/high sensitivity option?

Reply:
+1 for the sensitivity option. But that will require additional time and cooperation with the app. How about we leave the current values and we track this in a separate issue?

Reply:
Yes for sure, let's do that in a later stage!